client/auth/api.js

```diff
@@ -96,14 +96,12 @@ define([
      *   but the user is free to change it. Set to the string literal
      *   `blank` to ignore any previously signed in email. Default is
      *   the last email address used to sign in.
-     *   @param {String} [config.ui]
-     *   UI to present - `lightbox` or `redirect` - defaults to `redirect`
-     *   @param {Number} [options.zIndex]
+     *   @param {String} [config.ui="redirect"]
+     *   UI to present - `lightbox` or `redirect`
+     *   @param {Number} [config.zIndex=100]
      *   only used when `config.ui=lightbox`. The zIndex of the lightbox background.
-     *   @default 100
-     *   @param {String} [options.background]
+     *   @param {String} [config.background=rgba(0,0,0,0.5)]
      *   only used when `config.ui=lightbox`. The `background` CSS value
-     *   @default rgba(0,0,0,0.5)
```

Comment:
Turns out these `@default` weren't visible anywhere in the generated output (that I could see).